src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml

```diff
@@ -1,6 +1,375 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Rule Name="Build"
-      OverrideMode="Extend"
+      OverrideMode="Replace"
+      Description="Specifies properties that control how the project builds."
+      DisplayName="Compile"
+      PageTemplate="generic"
+      Order="200"
       xmlns="http://schemas.microsoft.com/build/2009/properties">
 
+  <Rule.Categories>
+    <Category Name="General"
+              DisplayName="General" />
+
+    <Category Name="CompileOptions"
+              DisplayName="Compile Options" />
+
+    <Category Name="WarningConfigurations"
+              DisplayName="Warning configurations" />
```

Comment:
Similarly, the label "Warnings" would suffice here IMO and prevent wrapping in the navigation tree.